src/components/SummaryDetail/SummaryDetail.js

```diff
@@ -24,7 +24,7 @@ const SummaryDetail = ({
         />
       </IconButton>
     )}
-    <div>
+    <div className="slds-container_fluid">
```

Comment:
This class is not present in the standalone `SummaryDetail`. We could add a `containerClassName` prop or similar to set it in `SetupAssistant`.

Reply:
An additional `containerClassName` prop sounds good! 👍